code/agent/config/dev_objects.json

```diff
@@ -5,14 +5,18 @@
       "id": "OpenDRIVE",
       "reading_frequency": 1
     },
+    {
+      "type": "sensor.pseudo.actor_list",
+      "id": "actor_list"
+    },
     {
       "type": "vehicle.lincoln.mkz_2020",
       "id": "hero",
       "role_name": "hero",
       "spawn_point": {
-        "x": 986.0,
-        "y": -5442.0,
-        "z": 371.0,
+        "x": 983.5,
+        "y": -5373.2,
```

Comment:
why do you need to change the spawn point of the car? do we want this change in main?